pyproject.toml

```diff
@@ -45,6 +45,12 @@ extend-exclude = [
     "_vendor",
     ".nox/",
 ]
+target-version = "py37"
```

Comment:
I'd set this via `project.requires-python` even though this then could have an influence on other tools, but it would be the "right influence": https://docs.astral.sh/ruff/settings/#target-version

Reply:
Looks like we are going to punt on this issue.

1. ruff doesn't support Python 3.6 so we can't make `target-version = "py36"`
2. We can make `project.requires-python = ">=3.6"` but ruff just internally bumps this to 3.7.
3. Specifying `[project]` requires specifying multiple fields
4. A full move to using pyproject for project metadata isn't possible yet (#3529).